src/lib/utils/options/readers/typedoc.ts

```diff
@@ -86,8 +86,12 @@ export class TypeDocReader implements OptionsReader {
             delete data["extends"];
         }
 
-        for (const [key, val] of Object.entries(data)) {
+        for (let [key, val] of Object.entries(data)) {
             try {
+                // The "packages" option is an array of paths and should be interpreted as relative to the typedoc.json
```

Comment:
This came up in testing because the tests cwd is not inside the project we're operating on, and without this the paths in the typedoc.json are interpreted relative to cwd, rather than to the file in which they appear.

This actually seems like something you might want to change for all options which are filepaths but could appear in the typedoc.json.

Reply:
Yes! I completely agree, #1442 tracks this, but last time I went after it the approach I took failed.